evm/get-duplicate-token-managers.js

```diff
@@ -0,0 +1,120 @@
+const { ethers } = require('hardhat');
+const { Contract, getDefaultProvider } = ethers;
+const info = require('../axelar-chains-config/info/mainnet.json');
+const tokenManagerInfo = require('../axelar-chains-config/info/tokenManagers.json');
+const ITokenManager = require('@axelar-network/interchain-token-service/artifacts/contracts/interfaces/ITokenManager.sol/ITokenManager.json');
+const IInterchainTokenService = require('@axelar-network/interchain-token-service/artifacts/contracts/interfaces/IInterchainTokenService.sol/IInterchainTokenService.json');
+const fs = require('fs');
+const toml = require('toml');
+
+const RPCs = toml.parse(fs.readFileSync('./axelar-chains-config/info/rpcs.toml', 'utf-8'));
+const itsAddresses = {
+    ethereum: '0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C',
+    avalanche: '0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C',
+    fantom: '0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C',
```

Comment:
switch to using the same cli setup and style like the other evm/sui scripts. makes the usage consistent along with supporting the same features and easier for others to understand, avoids hardcoding these values